### PR TITLE
Add Carousel Section

### PR DESCRIPTION
**Pull request by err53**

### CODE REVIEW

**Comment by Krish120003** [comment]
The mobile layout doesn't match the Figma design, missing padding around images and text/heading sizes mismatch the design.

**Review by Krish120003 — CHANGES_REQUESTED** [review]
screenshot

Why is there a placeholder box on each image? If this is the description, it needs to go below the images (refer to Figma).

The "Toast to one decade" title is much smaller than the design on desktop.

On mobile, the "One Decade" is very big and does not have sufficient padding (compare with Figma to see difference).
image


We have a pause button, is that required? It's very small and can be hard to tap on mobile.

When the image transitions, there is space used to the left of the < arrow, but the "current" image does not take up that space on desktop, but it does on mobile. Is this intentional/expected behaviour?

image
<details><summary>Browser metadata</summary>





```
Path:      /
Browser:   Chrome 118.0.0.0 on Mac OS 10.15.7
Viewport:  1492 x 892 @2x
Language:  en-US
Cookies:   Enabled
```

[Open in BrowserStack](https://www.browserstack.com/user/try-live?url=https%3A%2F%2Fdeploy-preview-12--elaborate-croquembouche-906522.netlify.app%2F&os=OS%20X&os_version=Catalina&browser=Chrome&browser_version=118.0&resolution=1492x892&speed=1&start=true&ref=netlify-source)



</details>

[Open Deploy Preview](https://deploy-preview-12--elaborate-croquembouche-906522.netlify.app/) · [Mark as Resolved](https://app.netlify.com/cdp/resolve?deployID=652b4b4a8a4f36000837b053&commentID=652d6c24985c733a771ac501&resolution=resolved)

**Comment by err53** [comment]
TL;DR
All review items will either take too long to fix or are unfixable given the current scope of work.

> Why is there a placeholder box on each image? If this is the description, it needs to go below the images (refer to Figma).

This is the carousel component's built in description component, if needed I can extract it to a separate component but adding animation to match the carousel will be messy. Not in scope.

> The "Toast to one decade" title is much smaller than the design on desktop.
> On mobile, the "One Decade" is very big and does not have sufficient padding (compare with Figma to see difference).

If I add padding the font size is incorrect, currently the font size matches the size of all other components. Given that the website is responsive, there is no way to satisfy perfect spacing **and** perfect font size on all screen sizes. #wontfix

> We have a pause button, is that required? It's very small and can be hard to tap on mobile.

It's a feature of the carousel library, useful for desktop accessibility. Again, since it's part of the library I can't resize it easily if I wanted to, and reimplementing it will take too long.

> When the image transitions, there is space used to the left of the < arrow, but the "current" image does not take up that space on desktop, but it does on mobile. Is this intentional/expected behaviour?

Probably a bug with the carousel library, #wontfix

**Review by Krish120003 — APPROVED** [review]
Looks good, needs polish but good for MVP. Needs proper images before prod.
